### ISSUE TITLE
Still blocked by CSP on github

### ISSUE THREAD

**Issue by brabalan** [mems/bookmarklets-context-menu#11]
I have a very simple bookmarklet that calls a protocol:
```
javascript:location.href='org-protocol:/capture?template=l&url='+encodeURIComponent(location.href)+'&title='+encodeURIComponent(document.title)+'&body='+encodeURIComponent(window.getSelection())
```

When I try to use it on github, I get this error:
```
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src https://assets-cdn.github.com”).
```

If I try to use it with the context menu, I now get this error:
```
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src https://assets-cdn.github.com”). Source: call to eval() or related function blocked by CSP.
```

Is this a bug?

**Comment by mems** [comment]
Yest it's related to CSP ([see](https://stackoverflow.com/questions/32062779/using-csp-content-security-policy-with-a-custom-protocol))

I'm investigating, but I think there is no viable solution to bypass this issue

**Comment by mems** [comment]
Fixed by ea9cb24b2fefa22e0331cbfc76bf89d6b84f6ae6

**Comment by brabalan** [comment]
It works great, thanks!
